make deploy error with message: ClusterRoleBinding "kfserving-proxy-rolebinding" is invalid

/kind bug

**What steps did you take and what happened:**
[A clear and concise description of what the bug is.]
make deploy-dev
or
make deploy

Error message "ClusterRoleBinding "kfserving-proxy-rolebinding" is invalid" reported

**What did you expect to happen:**
without error

**Anything else you would like to add:**
[Miscellaneous information that will assist in solving the issue.]

```
\# make deploy
go run vendor/sigs.k8s.io/controller-tools/cmd/controller-gen/main.go crd --output-dir=config/default/crds
CRD files generated, files can be found under path config/default/crds.
kustomize build config/default/crds -o config/default/crds/serving_v1alpha2_inferenceservice.yaml
go run vendor/sigs.k8s.io/controller-tools/cmd/controller-gen/main.go rbac --output-dir=config/default/rbac
RBAC manifests generated under 'config/default/rbac' directory
kustomize build config/default | kubectl apply -f -
namespace/kfserving-system configured
customresourcedefinition.apiextensions.k8s.io/inferenceservices.serving.kubeflow.org configured
clusterrole.rbac.authorization.k8s.io/kfserving-proxy-role configured
clusterrole.rbac.authorization.k8s.io/manager-role configured
clusterrolebinding.rbac.authorization.k8s.io/manager-rolebinding configured
configmap/inferenceservice-config configured
Warning: kubectl apply should be used on resource created by either kubectl create --save-config or kubectl apply
secret/kfserving-webhook-server-secret configured
service/kfserving-controller-manager-metrics-service unchanged
service/kfserving-controller-manager-service unchanged
statefulset.apps/kfserving-controller-manager unchanged
The ClusterRoleBinding "kfserving-proxy-rolebinding" is invalid: roleRef: Invalid value: rbac.RoleRef{APIGroup:"rbac.authorization.k8s.io", Kind:"ClusterRole", Name:"proxy-role"}: cannot change roleRef
make: *** [deploy] Error 1
```

--- a/Makefile
+++ b/Makefile
@@ -29,6 +29,7 @@ deploy: manifests
 
 deploy-dev: manifests
 	./hack/image_patch_dev.sh development
+	./hack/update-clusterrolebinding.sh config/overlays/development
 	kustomize build config/overlays/development | kubectl apply -f -
 
 deploy-local: manifests

--- a/hack/update-clusterrolebinding.sh
+++ b/hack/update-clusterrolebinding.sh
@@ -1,0 +1,34 @@
+#!/bin/bash
+
+# Copyright 2019 The Kubeflow Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Usage: update-clusterrolebinding.sh [CONFIG_DIR]
+
+CONFIG_DIR=$1
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+TMPFILE="/tmp/tmp_$$"
+
+if [ ! -d "${TMPFILE}" ] 
+then
+    mkdir ${TMPFILE}
+fi
+
+kustomize build ${CONFIG_DIR} -o ${TMPFILE}
+for i in ${TMPFILE}/rbac.authorization.k8s.io*clusterrolebinding*.yaml; do kubectl auth reconcile -f $i; done
+rm -rf ${TMPFILE}

